### PR TITLE
Copy file function implemented

### PR DESCRIPTION
**Pull request by thorium-zero**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,7 @@
-# write your code here
+from shutil import copy
+
+
+def copy_file(command: str) -> None:
+    comm_data = command.split()
```
Comment (YuriiKindrat) [inline app/main.py:5]: you can unpack these variables in the easiest way
`command, old_file, new_file = command.split()` 
so there is won't be need to use indexes later

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1 +1,28 @@
-# write your code here
+from shutil import copy
+from os.path import exists
+
+
+class ArgumentException(Exception):
+    pass
```
Comment (Oleg-s-s) [inline app/main.py:6]: I think its not necessary to create a custom exception. You can use ValueError and then combine em in except block

File: app/main.py [hunk 3]
```diff
@@ -1 +1,28 @@
...
+            raise ArgumentException("Destination must differ from the source")
+        if not exists(source):
+            raise FileNotFoundError(f"Unable to find the "
+                                    f"source file at '{source}'")
+    except ValueError:
+        print("You have to provide 3 parameters in the command string")
+    except ArgumentException as er:
+        print(f"Invalid arguments: {er}")
+    except FileNotFoundError as er:
+        print(er)
```
Comment (Oleg-s-s) [inline app/main.py:24]:
```suggestion
        if cmd != "cp":
            raise ValueError("Unknown command")
        if source == destination:
            raise ValueError("Destination must differ from the source")
        if not exists(source):
            raise FileNotFoundError(f"Unable to find the "
                                    f"source file at '{source}'")
    except (ValueError, FileNotFoundError) as error:
        print(f"Error: {error}")
```
Smth like this ^ What do you think?

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]

**Review by Oleg-s-s — APPROVED** [review]
:+1:

**Pull request closed without merging**